employees/factories.py

```diff
@@ -22,5 +22,6 @@ class Meta:
 class TaskActivityTypeFactory(factory.DjangoModelFactory):
     class Meta:
         model = TaskActivityType
+        django_get_or_create = ("name",)
```

Comment:
Is it needed here? We create `TaskActivityType` only with `name` field anyway. I think we can delete it

Reply:
It is needed as I need to enforce that only one task activity with a given name will be created. Without this most tests will fail as Factory will generete more than one task with a given name and condition on uniqnuess of task name will not be fullfield.